Fixes #1867 - Append date and time to admin notices

crystinutzaa opened the pull request:
Added a date & time after some messages related to: PreLoad, Clear Cache, CPCSS

GeekPress commented:
We will have to re-translate all the strings. As the date doesn't have any relation with the string, it could be added after like this:

```
$notice = sprintf( __( '%s: Cache cleared.', 'rocket' ), '<strong>' . WP_ROCKET_PLUGIN_NAME . '</strong>' );
$notice .= ' <em>' . date("Y-m-d h:i:sa") .'</em>';
```

Then, it will avoid us to have to translate the strings.

Also, a more user friendly format date could help like:

`date_i18n( "F j, Y @ G:i", time() )
`

crystinutzaa commented:
I split the dates from orginal messages and I changed the use of date_i18n function.
However I think that for PreLoad Complete Time I still need a transient.
Did I missed any messages which should contain the timestamp?
